README.md

```diff
@@ -68,31 +68,31 @@ First time kernel build
     ev3dev package repo and then run the `install-kernel-build-tools` script.
     (You only need to run this once.)
 
-        ~/work/ev3dev-buildscripts $ .sudo apt-add-repository http://ev3dev.org/debian
-        ~/work/ev3dev-buildscripts $ .sudo apt-key adv --keyserver pgp.mit.edu --recv-keys 2B210565
-        ~/work/ev3dev-buildscripts $ .sudo apt-get update
+        ~/work/ev3dev-buildscripts $ sudo apt-add-repository http://ev3dev.org/debian
+        ~/work/ev3dev-buildscripts $ sudo apt-key adv --keyserver pgp.mit.edu --recv-keys 2B210565
+        ~/work/ev3dev-buildscripts $ sudo apt-get update
         ~/work/ev3dev-buildscripts $ ./install-kernel-build-tools
 
 6.  Create a `local-env` to make use of all of your processing power. See the
     [Faster Builds and Custom Locations](#faster-builds-and-custom-locations)
     section below for more about this file.
 
-        ~/work/ev3dev-buildscripts $ echo "#!/bin/sh
+        ~/work/ev3dev-buildscripts $ echo "#\!/bin/sh
```

Comment:
I have to admit, I didn't check the meaningfulness, I just did make it work. However for me it works without the shbang.